src/test/java/com/SAS/acceptanceTests/TeamManagementAT.java

```diff
@@ -372,9 +373,76 @@ public void addAdditionalTeamOwnerFail() {
+            System.out.println(teamManagement.showOptionalNomineesForTeamManager(myTeam));
+            System.out.println('\n' + "Selected: Itay Cohen");
+            System.out.println("Would you like to give the new team manager editing assets privileges? 1 to give privileges, otherwise 0");
+            String approval = "1";
+            System.out.println(approval);
+            boolean givePrivileges = approval.equals("1") ? true : false;
+            User toMakeTeamOwner = teamManagement.getUserForTeamManagerNominees(myTeam, "Itay Cohen");
+            if (toMakeTeamOwner !=null){
+                toMakeTeamOwner = teamManagement.addTeamManager(toMakeTeamOwner, myTeam, teamOwner, givePrivileges);
+                assertTrue(toMakeTeamOwner instanceof TeamManager);
+                System.out.println(((TeamManager)toMakeTeamOwner).getFullName() + " was nominated as team manager");
+            }else{
+                System.out.println("The team manager could not be nominated");
+            }
+        }
+        else {
+            System.out.println("You do not have the privileges to add the team manager");
+        }
+    }
+
+    @Test
+    public void addTeamManagerFailWrongName() {
+        //preparations- create a player
+        User player = userController.createUser("ItayC", "ItAY1234", "Itay Cohen", UserType.PLAYER, true, null);
+        ((Player)player).setFieldRole(FieldRole.MIDFIELDER);
+        ((Player)player).setTeam(team);
+        ((Player)player).setDateOfBirth(LocalDate.parse("1990-12-01"));
+        team.addPlayerToTeam((Player)player);
+
+        Team myTeam = ((TeamOwner) teamOwner).getTeam();
+        myTeam.getPersonalPage().showPersonalPage();
+
+        //enter editing mode
+        if (teamManagement.enterEditingMode(teamOwner, myTeam)) {
+            System.out.println("--Select to add a team manager--");
+            System.out.println("Please select a nominee from the followings:");
+            System.out.println(teamManagement.showOptionalNomineesForTeamManager(myTeam));
+            System.out.println('\n' + "Selected: ");
+            System.out.println("Would you like to give the new team manager editing assets privileges? 1 to give privileges, otherwise 0");
+            String approval = "1";
+            System.out.println(approval);
+            boolean givePrivileges = approval.equals("1") ? true : false;
+            User toMakeTeamOwner = teamManagement.getUserForTeamManagerNominees(myTeam, "");
+            if (toMakeTeamOwner !=null){
+                toMakeTeamOwner = teamManagement.addTeamManager(toMakeTeamOwner, myTeam, teamOwner, givePrivileges);
+                assertTrue(toMakeTeamOwner instanceof TeamManager);
+                System.out.println(((TeamManager)toMakeTeamOwner).getFullName() + " was nominated as team manager");
+            }else{
+                System.out.println("The team manager could not be nominated");
```

Comment:
Missing assert of the team manager wasn't nominated - as:
assertNull(team.getTeamManager());

Reply:
I don't think we should put more that one assert as it supposed to check only what I expect to happen..